### PR TITLE
Prepare V1.9.0

### PR DESCRIPTION
**Pull request by megawac**
Theres been a number of changes in terms of features and support. I think its time for an updated release.

Prerequisite issues
- [ ] #2212
- [ ] #2214 
- [ ] #2073 
- [ ] #2071 
- [x] #2211 
- [ ] #2231

/cc @jridgewell @jashkenas @akre54

### CODE REVIEW

**Comment by jdalton** [comment]
Ping.

**Comment by jridgewell** [comment]
I think the only showstopper here is #2214.

File: index.html [hunk 1]
```diff
@@ -1170,6 +1175,25 @@ <h2 id="functions">Function (uh, ahem) Functions</h2>
 });
 </pre>
 
+<p id="restParam">
+        <b class="header">memoize</b><code>_.memoize(function, [index=function.length-1])</code>
```
Comment (jridgewell) [inline index.html]: Memoize? :stuck_out_tongue:

File: index.html [hunk 2]
```diff
@@ -1170,6 +1175,25 @@ <h2 id="functions">Function (uh, ahem) Functions</h2>
...
+        Similar to ES6's <a href="https://developer.mozilla.org/docs/Web/JavaScript/Reference/Functions/rest_parameters">rest param</a>,
+        <code>restParam</code> collects arguments passed to a function from
+        a given <code>index</code> as an array.
+      </p>
+      <pre>
+function multiply(multiplier, ...theArgs) {
+  return theArgs.map(function (element) {
+    return multiplier * element;
+  });
+}
```
Comment (jridgewell) [inline index.html]: It'd probably be more helpful to show how a `restParam` function translates into ES6.

File: index.html (hunk 1 above)
Comment (megawac) [inline index.html]: Whoops (feel free to just push to this branch)

**Comment by jgonggrijp** [comment]
Closing this as we are already past version 1.9.0.

**Pull request closed without merging**